### DISCUSSION TITLE
How to disable launch_menu completely ?

### DISCUSSION THREAD

**Discussion by volgar1x** [wezterm/wezterm#564] [Q&A] [answered]
I use tmux and the launch menu is redundant

**Comment by wez** [comment, marked as answer]
The launch menu is only shown via hot key or right clicking on the "+" button in the tab bar.

If you want to disable the tab bar, you can set `enable_tab_bar = false`:

https://wezfurlong.org/wezterm/config/appearance.html?highlight=enable_tab_bar#tab-bar-appearance--colors
